[EN/KO] Update Medals (#2958)
ppy/osu-wiki: wiki/Medals/en.md

The Combo medals can be obtained by gaining a specific [combo](/wiki/combo) in osu!standard and can be completed with any one song. Achieving the same combo count in [osu!taiko](/wiki/osu!taiko), [osu!catch](/wiki/osu!catch), or [osu!mania](/wiki/osu!mania) mode will not result in getting the medal.

> For the 1,000 and 2,000 combo medals, you should find long beatmaps. **If the beatmap is not in ranked or approved state, it's not recognized.**

Reviewer: ```suggestion
For the 1,000 and 2,000 combo medals, you should look for sufficiently long beatmaps. **If the beatmap is not in ranked or approved state, the medals will not be granted.**
```

> For the 1,000 and 2,000 combo medals, you should look for sufficiently long beatmaps. **If the beatmap is not in ranked or approved state, the medals will not be granted.**

Reviewer 2: also capitalize Ranked/Approved

> For the 1,000 and 2,000 combo medals, you should look for sufficiently long beatmaps. **If the beatmap is not in ranked or approved state, the medals will not be granted.**

Reviewer 3: the suggestion was updated

---

The High Ranker medals can be obtained by reaching a certain global ranking position in **any** game mode.

---

> The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing any map of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be obtained if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the map will round to the nearest star difficulty and will give the medal for what it rounds to. For example, if you haven't earned 6/7 star medals but clear 7.8 stars, you'll only get 7 star medals. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The map must also be ranked to be able to earn a medal.

Reviewer: ```suggestion
The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing a beatmap of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be granted if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the map will be rounded to the nearest star difficulty and will grant the medal for what it is rounded to. For example, if you haven't earned 6/7 star medals but cleared 7.8 stars difficulty, you will only receive a 7 star medal. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The beatmap must also be ranked to grant a medal.
```

> The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing a beatmap of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be granted if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the map will be rounded to the nearest star difficulty and will grant the medal for what it is rounded to. For example, if you haven't earned 6/7 star medals but cleared 7.8 stars difficulty, you will only receive a 7 star medal. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The beatmap must also be ranked to grant a medal.

Reviewer 3: ```suggestion
The star-based medals are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing a beatmap of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be granted if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the beatmap will be rounded to the nearest star difficulty and will grant the medal for what it is rounded to. For example, if you have not earned the 6/7 star medals but cleared 7.8 stars difficulty, you will only receive a 7 star medal. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The beatmap must also be ranked to grant a medal.
```

- Lowercase star-based
- achivements (typo) -> medals
- map -> beatmap
- haven't -> have not

Reviewer 3: I'm a bit confused with the example about not have had earned the 6/7 star medals(?) but completing a 7.8 star medal map will give you the 7 star medal?

Author: I think it doesn't earn 6, 7 stars medal at the same time. But I've never done that before, so I'm not sure :thinking:

Reviewer 2: the point there is that getting the 7 star medal doesn't also give you the 6 star one--- this should probably be explained in a way that's not just a random example

Author: I think we can just delete the example section
